### PR TITLE
docs: Jans readme updates

### PR DESCRIPTION
**Pull request by ossdhaval**
### Prepare

- [x] Read contribution guidelines
- [x] Read license information

-------------------

### Description

README should have more details to describe and promote Janssen in a better way in the user base and community

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,142 @@
+<h1 align="center"><img src="https://github.com/JanssenProject/jans/blob/eea33ea5c7740244b45f7bb1893167cd3fb93903/docs/logo/janssen_project_transparent_630px_182px.png" alt="Janssen Project - cloud native identity and access management platform"></h1>
+
+## Welcome to the Janssen Project
+
+Janssen enables organizations to build a scalable centralized authentication and authorization service using free open source software. The components of the project include client and server implementations of the OAuth, OpenID Connect, SCIM and FIDO standards. 
+
+**Releases**: [Latest](https://github.com/JanssenProject/jans/releases/latest) | [All](https://github.com/JanssenProject/jans/releases)
+
+**Get Help**: [Chat](https://gitter.im/JanssenProject/Lobby) | [Discussions](https://github.com/JanssenProject/jans/discussions)
```
Comment (moabu) [inline README.md]: Lets put the discussions first.

File: .github/SECURITY.md [hunk 2]
```diff
@@ -14,8 +14,7 @@ Security updates will typically only be applied to the latest release (at least
 
 ## Reporting a vulnerability
 
-To report a security issue, email [security@jans.io](mailto:security@jans.io?subject=SECURITY)
-and include the word "SECURITY" in the subject line.
+To report a security issue, send an email to [security@jans.io](mailto:security@jans.io?subject=SECURITY)
```
Comment (moabu) [inline .github/SECURITY.md:17]: is this email working ?

**Review by moabu — CHANGES_REQUESTED** [review]

File: .github/SECURITY.md (hunk 2 above)
Comment (shmorri) [reply .github/SECURITY.md:17]: It looks like this is from a find/replace for gluu.org -> jans.io .

File: README.md (hunk 1 above)
Comment (ossdhaval) [reply README.md]: Fixed with [a72b528](https://github.com/JanssenProject/jans/pull/1055/commits/a72b52838030607e484eeeed2ea372b57c10bb05)

File: .github/SECURITY.md (hunk 2 above)
Comment (ossdhaval) [reply .github/SECURITY.md:17]: Oh no no. Not a find/replace. 
@ganesh-at-wiw had created this email long back on my request. And as far as I can remember, we had tested it as well. I have asked @ganesh-at-wiw to confirm if it is still working.

Comment (ossdhaval) [reply .github/SECURITY.md:17]: Tested again. Mail ID is working.

**Review by moabu — APPROVED** [review]

**Review by nynymike — APPROVED** [review]
An improvement...

**Pull request merged**